src/lua/utils.c

```diff
@@ -121,6 +121,12 @@ luaL_pushuuidstr(struct lua_State *L, const struct tt_uuid *uuid)
 	lua_pushlstring(L, str, UUID_STR_LEN);
 }
 
+struct datetime *
+luaL_pushdatetime(struct lua_State *L)
```

Comment:
AFAIR, the agreement was to use `luaT` for tarantool-specific types. @igormunkin knows more.

Reply:
Do you realize that not properly documented agreement (in the SOP and code- guidelines) is actually not existing (at least, for those not taking participation in past discussion)? Sort of imaginary rules. I could not find it easily neither in SOP, nor in code guidelines, nor in any Wiki page, may be I looked not very much carefully.

But now you've mentioned it and I've managed to find this discussion - https://github.com/tarantool/tarantool/issues/4577#issuecomment-778457513 could anyone of involved in that discussion publish it finally in usual locations where developer read code guidelines? @igormunkin ?

P.S.
Will rename to `luaT_pushdatetime`

Reply:
I totally realize. My rules and agreements are mostly imaginary, I see it now. You can use any name here.